persistent_connection.go

```diff
@@ -228,6 +242,48 @@ func (pconn *persistentConn) hasFeatureWithArg(name, arg string) bool {
 	return found && strings.ToUpper(arg) == val
 }
 
+func (pconn *persistentConn) setClient() error {
+        code, msg, err := pconn.sendCommand("CLNT GoFTP")
```

Comment:
According to the RFCs - it has no real purpose, just telling the server what client you are using.
But as I was testing I found a few servers that would not allow "OPTS UTF8 ON" unless CLNT is issued before...

For example on some version of CGFTP:
OPTS UTF8 ON
500 Send 'CLNT client_type' before enabling UTF8